quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java

```diff
@@ -327,6 +337,8 @@ void persistBuildTimeProperties(BuildProducer<GeneratedResourceBuildItem> resour
             properties.put(String.format("kc.provider.file.%s.last-modified", jar.getName()), String.valueOf(jar.lastModified()));
         }
 
+        properties.put(QUARKUS_PROPERTY_ENABLED, String.valueOf(QuarkusPropertiesConfigSource.getConfigurationFile() != null));
```

Comment:
Might be more reusable and readable to use e.g. a method like `quarkusPropertiesExists()` in `QuarkusPropertiesConfigSource` which hides the String.valueOf... implementation detail? 

Advantage:
* ConfigSource related impl stays in ConfigSource class, can be reused elsewhere and when it changes you only have to change it there

Reply:
I see. I would leave that way.